priv/libexec/commands/win/console.ps1

```diff
@@ -1,12 +1,12 @@
 ## Run the app in console mode
 $bin = whereis-erts-bin
-$werl = (join-path $bin werl)
+$erl = (join-path $bin "erl.exe") #get erl.exe as werl.exe will open a new window
 
 $boot = (join-path $Env:REL_DIR $Env:REL_NAME)
 
-$argv = @("-boot", $boot)
-$argv += @("-config", $Env:SYS_CONFIG_PATH)
-$argv += @("-args_file", $Env:VMARGS_PATH)
+$argv = @("-boot", "`"$boot`"")
```

Comment:
Just for my own edification, could you explain the changes here? I'm going to merge them, I'd just like to know what I need to be doing differently or how I should be reasoning about quoting here; I'm working with PS Core for the most part, and I know there are differences with previous versions of PS, but from a maintenance standpoint I can't work with both, so Core is what I've stuck with, do these changes make any assumptions in that regard?

Reply:
so, the difference here is that we wrap the `$boot` in quotes so that paths containing a space work as intended, rather than acting as independent args 👍